Building from release does not work in Ubuntu 20.04

# Required information

 * Distribution: github release tarball
 * Distribution version: 4.12

# Issue description

make fails with "go install: no install location for directory..."

# Steps to reproduce

1. Download lxd-4.12.tar.gz
2. Unpack
3. Follow building from release instrucitons in README.md. Everything works up to make.
...
make
go get -t -v -d ./...
CC=cc go install -v -tags "libsqlite3"  ./...
go install: no install location for directory /root/lxd-4.12/fuidshift outside GOPATH
        For more details see: 'go help gopath'
go install: no install location for directory /root/lxd-4.12/lxc outside GOPATH
        For more details see: 'go help gopath'
go install: no install location for directory /root/lxd-4.12/lxc-to-lxd outside GOPATH
        For more details see: 'go help gopath'
go install: no install location for directory /root/lxd-4.12/lxd outside GOPATH
        For more details see: 'go help gopath'
go install: no install location for directory /root/lxd-4.12/lxd/db/generate outside GOPATH
        For more details see: 'go help gopath'
go install: no install location for directory /root/lxd-4.12/lxd-agent outside GOPATH
        For more details see: 'go help gopath'
go install: no install location for directory /root/lxd-4.12/lxd-benchmark outside GOPATH
        For more details see: 'go help gopath'
go install: no install location for directory /root/lxd-4.12/lxd-p2c outside GOPATH
        For more details see: 'go help gopath'
go install: no install location for directory /root/lxd-4.12/test/deps outside GOPATH
        For more details see: 'go help gopath'
go install: no install location for directory /root/lxd-4.12/test/macaroon-identity outside GOPATH
        For more details see: 'go help gopath'
make: *** [Makefile:20: default] Error 1

--- a/Makefile
+++ b/Makefile
@@ -8,6 +8,7 @@ ARCHIVE=lxd-$(VERSION).tar
 HASH := \#
 TAG_SQLITE3=$(shell printf "$(HASH)include <dqlite.h>\nvoid main(){dqlite_node_id n = 1;}" | $(CC) ${CGO_CFLAGS} -o /dev/null -xc - >/dev/null 2>&1 && echo "libsqlite3")
 GOPATH ?= $(HOME)/go
+export GO111MODULE=off
 
 .PHONY: default
 default:

